### PR TITLE
WIP: Test cherry picks

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
I am waiting on #881 to be merged in before merging this in.

This should probably go in before #880

### CODE REVIEW

File: module/PowerShellEditorServices/PowerShellEditorServices.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 RootModule = 'PowerShellEditorServices.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.10.3'
+ModuleVersion = '1.11.0'
```
Comment (rkeithhill) [inline module/PowerShellEditorServices/PowerShellEditorServices.psd1]: This will get bumped to `1.11.1` before the release, right?

Comment (TylerLeonhardt) [reply module/PowerShellEditorServices/PowerShellEditorServices.psd1]: Yeah it will be 1.12.0 because weve got features!

Comment (rkeithhill) [reply module/PowerShellEditorServices/PowerShellEditorServices.psd1]: Even better.

**Comment by TylerLeonhardt** [comment]
Yeah this will be closed and I will do it to the branch directly so we keep the full history.

**Comment by TylerLeonhardt** [comment]
this should go into legacy

**Pull request closed without merging**